package.json

```diff
@@ -197,6 +196,7 @@
     "stylis-plugin-rtl": "^1.0.0",
     "terser-webpack-plugin": "^4.2.3",
     "thread-loader": "^3.0.4",
+    "query-string": "^7.0.1",
```

Comment:
I thought the plan was to remove it, not move to dev dependencies?

(Aside: it's not listed in alphabetical order now)

Reply:
It is. This is a draft. I need to workout this - https://github.com/google/web-stories-wp/pull/8640#issuecomment-895954084